Sources/SPTPersistentCache.m

```diff
@@ -61,6 +61,13 @@ void SPTPersistentCacheSafeDispatch(_Nullable dispatch_queue_t queue, _Nonnull d
     }
 }
 
+@interface SPTPersistentCacheFileInfo : NSObject
+@property (nonatomic, strong, readonly) NSString *fileName;
+@property (nonatomic, strong, readonly) NSDate *mdate;
+@property (nonatomic, assign, readonly) off_t fileSize;
+- (instancetype)initWithFileName:(NSString *)fileName mdate:(NSDate *)mdate fileSize:(off_t)fileSize;
+@end
```

Comment:
previously using a messy dictionary structure for what amounted to 3 properties (likely no worse performance-wise than nested dictionaries)